holidays/countries/benin.py

```
@@ -0,0 +1,166 @@
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.groups import ChristianHolidays, InternationalHolidays, IslamicHolidays
+from holidays.observed_holiday_base import ObservedHolidayBase, FRI_TO_PREV_THU
```

Ruff (0.11.9)

`holidays.observed_holiday_base.FRI_TO_PREV_THU` imported but unused

Remove unused import: `holidays.observed_holiday_base.FRI_TO_PREV_THU`

(F401)